### PR TITLE
[DB/Script] Mulgore - Camp narache cosmetics

### PR DESCRIPTION
**Pull request by Cidragon**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [x] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:
https://github.com/HavenWoW/BFA-HavenCore/issues/549

## SOURCE:

The changes have been validated through:
- [x] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. Check for NPC Brave "Windfeather" (3209) for repeatable paths.
2. Check for the battle between Fledgling Brave and Bristleback Invader where non of them should die between each other but players are allowed to kill them.
3. Fledgling Brave who are dead and with the fire aura shouldn't be selectable.

### CODE REVIEW

File: src/server/scripts/Kalimdor/zone_mulgore.cpp [hunk 1]
```diff
@@ -211,10 +211,244 @@ class spell_mulgore_funeral_offering : public SpellScript
     }
 };
 
+// NPC Fledgling Brave 36942
+class npc_fledgling_brave : public CreatureScript
```
Comment (Xenphis) [inline src/server/scripts/Kalimdor/zone_mulgore.cpp]: Avoid to use old registry class and move to RegisterCreatureAI.
